code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -1236,6 +1250,11 @@ pub mod pallet {
 			let mut total_weight = unix_time_now_weight;
 
 			RewardPools::<T>::translate(|pool_id, mut reward_pool: RewardPoolOf<T>| {
+				// If reward pool has not started, do not accumulate rewards or adjust weight
+				if reward_pool.start_block > frame_system::Pallet::<T>::current_block_number() {
```

Comment:
```suggestion
				if reward_pool.start_block >= frame_system::Pallet::<T>::current_block_number() {
```
?

Reply:
No, this checks to see if we should *not* accumulate rewards. If we are on or past the `start_block`, we should accumulate. Correct? @vimukthi-git @benluelo

Reply:
yeah this is correct, I might consider inverting the check though:

```
if start_block < current block {
  // current closure body
} else {
  Some(reward_pool)
}
```

Reply:
hahaha you got me